src/app/commissioner_app.cpp

```diff
@@ -1084,6 +1084,20 @@ bool CommissionerApp::JoinerKey::operator<(const JoinerKey &aOther) const
     return mType < aOther.mType || (mType == aOther.mType && mId < aOther.mId);
 }
 
+CommissionerDataset CommissionerApp::MakeDefaultCommissionerDataset()
+{
+    CommissionerDataset dataset;
+
+    dataset.mJoinerUdpPort = kDefaultJoinerUdpPort;
+    dataset.mPresentFlags |= CommissionerDataset::kJoinerUdpPortBit;
+    dataset.mAeUdpPort = kDefaultAeUdpPort;
+    dataset.mPresentFlags |= CommissionerDataset::kAeUdpPortBit;
+    dataset.mNmkpUdpPort = kDefaultNmkpUdpPort;
+    dataset.mPresentFlags |= CommissionerDataset::kNmkpUdpPortBit;
```

Comment:
For the default, add condition to only include AE UDP port and NMKP port when CCM mode?